### PR TITLE
fun-apis-blog

### PR DESCRIPTION
**Pull request by texas-kristen**
Posting https://docs.google.com/document/d/19_cWRWiv9U2mOT10myRXyMChhI2kujNFwqLpHrZXA9k/edit

### CODE REVIEW

**Comment by lottamus** [comment]
Deploy preview for *stoplightio* ready!

Built with commit 972b20085b91478231c7cb2ca13564e1f3ffe924

https://deploy-preview-1475--stoplightio.netlify.app

File: netlify/blog-posts/just-a-simple-api.md [hunk 1]
```diff
@@ -0,0 +1,312 @@
...
+	"name": "Blinded",
+	"desc": [
+		"- A blinded creature can't see and automatically fails any ability check that requires sight.",
+		"- Attack rolls against the creature have advantage, and the creature's attack rolls have disadvantage."
+	],
+	"url": "/api/conditions/blinded"
+}
+```
+
+If Dungeons and Dragons isn’t your game, that’s okay. Another open API with great documentation is the **[PokéAPI](https://pokeapi.co/docs/v2.html)**, an API for Pokemon data. There’s an incredible amount of Pokemon data, and PokeAPI has a lot of it, so it may be useful to limit the data. For a short example, let’s just look at the first two entries under the “generations” endpoint:
```
[the remark this answers is not in the document]
Comment (ksatx) [reply netlify/blog-posts/just-a-simple-api.md:99]: Ah, it's supposed to be [https://pokeapi.co/docs/v2](https://pokeapi.co/docs/v2), good catch

File: netlify/blog-posts/just-a-simple-api.md [hunk 2]
```diff
@@ -0,0 +1,312 @@
...
+  	"vel"
+   ],
+   "signature" : {
+  	"source" : "NASA/JPL Fireball Data API",
+  	"version" : "1.0"
+   }
+}
+```
+
+If a collection of meteor data is too numerically-dense for you, but you’re still fascinated by space, there’s also the **[Open Notify API](http://open-notify.org/) **. It’s an API that lets you pull some simple space-related data, such as the location of the ISS and the number of people in space at the current moment. How many people are in space right now? Let’s see:
```
Comment (paulatulis) [inline netlify/blog-posts/just-a-simple-api.md:238]:
```suggestion
If a collection of meteor data is too numerically-dense for you, but you’re still fascinated by space, there’s also the **[Open Notify API](http://open-notify.org/)**. It’s an API that lets you pull some simple space-related data, such as the location of the ISS and the number of people in space at the current moment. How many people are in space right now? Let’s see:
```

File: netlify/blog-posts/just-a-simple-api.md [hunk 3]
```diff
@@ -0,0 +1,312 @@
...
+  favicon: /images/mark_light_bg.png
+  robots: 'index, follow'
+  title: >-
+    Just a Simple API: List of APIs with Fun Data for Quick Mock-ups
+  image: /images/roller-coaster.jpg
+  twitter:
+    title: >-
+      Just a Simple API: List of APIs with Fun Data for Quick Mock-ups
+    image: /images/roller-coaster.jpg
+	username: '@stoplightio'
```
Comment (philsturgeon) [inline netlify/blog-posts/just-a-simple-api.md:36]: YAML error
```suggestion
    username: '@stoplightio'
```

**Pull request merged**